openapi/components/schemas/component.yaml

```
@@ -0,0 +1,24 @@
+allOf:
+  - $ref: ./shallowItem.yaml
+  - type: object
+  - required:
+    - imageName
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)